From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Mon, 9 May 2016 17:05:53 +0200
Subject: [PATCH] UX: Update bootstrap/target build time estimate.

While building for a different architecture (e.g. AArch64 on x86_64),
the bootstrap/target image build may take longer, due to latency
introduced by using qemu-user-static.

Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
---
 build/bootstrap_admin_node.sh.patch | 23 +++++++++++++++++++++++
 1 file changed, 23 insertions(+)

diff --git a/build/bootstrap_admin_node.sh.patch b/build/bootstrap_admin_node.sh.patch
index c66b497..46b53e6 100644
--- a/build/bootstrap_admin_node.sh.patch
+++ b/build/bootstrap_admin_node.sh.patch
@@ -1,6 +1,29 @@
 *** bootstrap_admin_node.sh.orig	Mon May 30 06:31:38 2016
 --- bootstrap_admin_node.sh	Mon May 30 06:35:11 2016
 ***************
+*** 86,94 ****
+  http://docs.openstack.org/developer/fuel-docs/userdocs/fuel-install-guide/bootstrap/\
+  bootstrap_troubleshoot.html"
+  bs_progress_message="There is no active bootstrap. Bootstrap image building \
+! is in progress. Usually it takes 15-20 minutes. It depends on your internet \
+! connection and hardware performance. After bootstrap image becomes available, \
+! reboot nodes that failed to be discovered."
+  bs_done_message="Default bootstrap image building done. Now you can boot new \
+  nodes over PXE, they will be discovered and become available for installing \
+  OpenStack on them"
+--- 86,96 ----
+  http://docs.openstack.org/developer/fuel-docs/userdocs/fuel-install-guide/bootstrap/\
+  bootstrap_troubleshoot.html"
+  bs_progress_message="There is no active bootstrap. Bootstrap image building \
+! is in progress. Usually it takes 15-20 minutes for a native build (x86_64) \
+! and/or 30-45 minutes for each cross-build (e.g. AArch64). \
+! It depends on your internet connection, hardware performance and selected \
+! bootstrap architecture(s). This ISO supports AArch64 only. After bootstrap \
+! image becomes available, reboot nodes that failed to be discovered."
+  bs_done_message="Default bootstrap image building done. Now you can boot new \
+  nodes over PXE, they will be discovered and become available for installing \
+  OpenStack on them"
+***************
 *** 339,346 ****
   set +x
   echo "Done!"
